.github/workflows/validate.yml

```diff
@@ -54,7 +54,6 @@ jobs:
         with:
           eslint: true
           stylelint: true
-        auto_fix: true
```

Comment:
any reason to remove this?

Reply:
Actually, this option was causing a problem in the workflow, so it's okay to remove it.